win32.c

```diff
@@ -7,6 +7,10 @@
 #define ENET_BUILDING_LIB 1
 #include "enet/enet.h"
 #include <windows.h>
+#ifdef __MINGW32__
+typedef UINT32 QOS_FLOWID, *PQOS_FLOWID;
```

Comment:
Mingw-w64 seems to already have `QOS_FLOWID` defined.

https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-headers/include/qos2.h#l141

Reply:
For `*PQOS_FLOWID` it's still missing. So what I need should be adding the latter one only, right?